creusot/src/translation/function.rs

```diff
@@ -879,11 +927,11 @@ impl<'tcx> TranslationCtx<'tcx> {
             let self_ = Term::var(Symbol::intern("self"), env_ty);
```

Comment:
This is the second definition of `self_`, so this one can be removed (and the `let mut csubst` just below can be moved inside the if to avoid a borrow checker error)